charts/reform-scan-blob-router/values.yaml

```diff
@@ -27,6 +27,8 @@ java:
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
     REJECT_DUPLICATES_CRON: "0/30 * * * * *"
+    CHECK_NEW_ENVELOPES_CRON: "0 55 0 * * MON-FRI" # every hour at 55 minutes on weekdays
+    CHECK_NEW_ENVELOPES_TTL: PT1H
```

Comment:
what's that?

Reply:
This will be used here: https://github.com/hmcts/blob-router-service/blob/20fb4707aef03ea11ed5ad61ff3e53b59fb7ac4f/src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java#L50
If we can always check for the last 1 hour, this config can be removed

Reply:
This value remains the same in all environments, so I'll rename this add this to `aplication.yaml`.